src/extensions/nativeExtensions/image/ImageWindow.vue

```diff
@@ -46,26 +47,34 @@
                     </v-icon>
                   </v-btn>
                 </v-img>
+                <v-text-field
+                  v-model="source.alt"
+                  label="Alt Text"
+                />
               </v-col>
             </v-row>
           </div>
         </v-expand-transition>
-
-        <v-text-field
-          v-model="form.src"
-          :label="$i18n.getMsg('extensions.Image.window.form.sourceLink')"
-        />
-
-        <v-expand-transition>
-          <div v-show="!form.src">
-            <div class="text-center grey--text mt-2 mb-4">
-              {{ $i18n.getMsg('extensions.Image.window.or') }}
-            </div>
-
-            <ImageUploadArea @select-files="onFilesSelect" />
-          </div>
-        </v-expand-transition>
       </v-card-text>
+      <v-tabs fixed-tabs>
+        <v-tab
+          v-for="(imageTab, i) in imageTabs"
+          :key="'tab-' + i"
+          :href="'#tab-' + i"
+        >
+          {{ imageTab.name }}
+        </v-tab>
+        <v-tab-item
+          v-for="(imageTab, i) in imageTabs"
+          :key="'tab-item-' + i"
+          :value="'tab-' + i"
+        >
+          <component
+            :is="imageTab.component"
+            @select-file="onFileSelect"
+          />
+        </v-tab-item>
+      </v-tabs>
```

Comment:
I wonder if this will work?

```suggestion
      <template v-for="(imageTab, i) in imageTabs">
        <v-tab
          :key="'tab-' + i"
          :href="'#tab-' + i"
        >
          {{ imageTab.name }}
        </v-tab>
        <v-tab-item
          :key="'tab-item-' + i"
          :value="'tab-' + i"
        >
          <component
            :is="imageTab.component"
            @select-file="onFileSelect"
          />
        </v-tab-item>
      </v-tabs>
    </template>
```

Reply:
Took me a minute to see what you've changed - only 1 iterator. Might work but it's not what the [Vuetify documentation](https://vuetifyjs.com/en/components/tabs/) is demonstrating.

Reply:
I should have indicated what I changed, sorry. I believe that this way the code repeats less, one cycle is better than two.